themes/digital.gov/layouts/partials/core/get_authors.html

```diff
@@ -35,7 +35,8 @@
           <div class="grid-col-9">
             <div class="details">
               <h5>
-                <a href="{{- (printf "authors/%s/" $uid) | absURL -}} " title="Posts by {{- $author.Params.display_name | default $uid -}} " rel="author">
+                <span class="usa-sr-only">Orginally posted by&nbsp;{{- $author.Params.display_name | default $uid -}}&nbsp;on&nbsp;{{- $.Date.Format "Jan 2, 2006" -}}</span>
```

Comment:
If this is for screen readers only, we can remove the non-breaking spaces.

Reply:
Got it, on my voice over it still reads it correctly without the spaces.